app/src/main/java/com/droidhats/campuscompass/views/MapFragment.kt

```diff
@@ -55,11 +50,13 @@ import com.google.android.libraries.places.widget.model.AutocompleteActivityMode
 import com.google.android.material.bottomsheet.BottomSheetBehavior
 import com.google.android.material.navigation.NavigationView
 import com.google.maps.android.PolyUtil
-import kotlinx.android.synthetic.main.bottom_sheet_layout.bottom_sheet
+import kotlinx.android.synthetic.main.bottom_sheet_layout.*
```

Comment:
same, for this import statement, I will add the specific import in my commit

Reply:
Fixed! 👍